stable/traefik/templates/service-monitor.yaml

```diff
@@ -0,0 +1,28 @@
+{{- if and .Values.metrics.prometheus.enabled .Values.metrics.prometheus.serviceMonitor.enabled }}
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
```

Comment:
Should we add the annotations here?

Reply:
I don't think annotations are necessary for the service monitor.